tools/src/test/java/org/cyclonedx/schema/BaseSchemaVerificationTest.java

```diff
@@ -29,6 +29,7 @@ List<String> getAllResources() throws Exception {
         files.addAll(getResources("1.3/"));
         files.addAll(getResources("1.4/"));
         files.addAll(getResources("1.5/"));
+        files.addAll(getResources("1.6/"));
```

Comment:
this file seams to be slipped in by accident. it is irrelevant for the java upgrade.
please remove it and create an extra PR for it

Reply:
For this particular file, I can create a third PR _[FEATURE] Adding tests for CycloneDX 1.6_.

This PR will only add this line `files.addAll(getResources("1.6/"));`.

Reply:
I removed `files.addAll(getResources("1.6/"));.` from file `tools/src/test/java/org/cyclonedx/schema/BaseSchemaVerificationTest.java`.

But I will wait for #480  to be merged before adding back 1.6 tests.

Reply:
Agreed :smiley: . However, I will wait for the PR #480 to be merged, because it will be easier for me to do the PR dedicated for 1.6 after the java lib update is merged.
Otherwise I need to revert to many commits on my end.

Reply:
understood. will merge this PR soon.